qt-models/diveplannermodel.cpp

```diff
@@ -130,7 +132,8 @@ void DivePlannerPointsModel::loadFromDive(dive *d)
 		}
 	}
 	// make sure we get the last point right so the duration is correct
-	if (!hasMarkedSamples) addStop(0, d->dc.duration.seconds,cylinderid, last_sp.mbar, true);
+	if (!hasMarkedSamples && !dc->last_manual_time.seconds)
+			addStop(0, d->dc.duration.seconds,cylinderid, last_sp.mbar, true);
```

Comment:
this has two tabs instead of one before `addStop`.

Reply:
done